src/Sulu/Component/Content/Tests/Unit/Document/Subscriber/FallbackLocalizationSubscriberTest.php

```diff
@@ -132,14 +128,9 @@ public function testNoWebspaceDoNotLoadGhostContent()
     {
         $this->inspector->getWebspace($this->document->reveal())->willReturn(null);
         $this->inspector->getLocales($this->document)->willReturn(['de', 'fr']);
-        $this->hydrateEvent->setLocale('de')->shouldBeCalled();
+        $this->hydrateEvent->setLocale('de')->shouldNotBeCalled();
```

Comment:
I get that `updateLocale` shouldn't be called anymore, but shouldn't it still change the localization of the event, so that the next subscribers take the right language?